src/components/CityColumnChart/CityColumnChart.js

```diff
@@ -365,21 +364,11 @@ class CityColumnChart extends Component {
           </header>
 
           <div className={style.metaWrapper}>
-            <Tooltip
-              borderColor={colorMedium}
-              text={firstIndicator.contextual
-                ? STRINGS.CONTEXTUAL_DEFINITION
-                : STRINGS.PERFORMANCE_DEFINITION
-              }
-            >
-              <Icon
-                className={style.indicatorTypeMark}
-                color={colorDark}
-                icon={firstIndicator.contextual ? 'contextualIndicator' : 'performanceIndicator'}
-                size={14}
-                title=""
-              />
-            </Tooltip>
+            <IndicatorIcon
+              className={style.indicatorTypeMark}
+              color={getColorVariant(props.colorBase, '500')}
+              contextual={firstIndicator.contextual}
+            />
```

Comment:
There were no visible changes to the chart, this just abstracts the "C" or "P" icon wrapped in a tooltip to a new `IndicatorIcon` component.